tobac/feature_detection.py

```diff
@@ -1235,11 +1242,14 @@ def feature_detection_multithreshold(
     """
     from .utils import add_coordinates, add_coordinates_3D
 
+    time_var_name: str = "time"
```

Comment:
I have deliberately not done this for now as we don't really have support for it throughout. It would take a pretty substantial refactor (I think) to ensure support all the way through.